webpack/assets/javascripts/foreman_tools.js

```diff
@@ -134,6 +134,9 @@ export function updateTable(element) {
     }
     uri.setSearch('per_page', perPage);
   }
+  if ($('#search-form').find('.autocomplete-input').val().trim() === '') {
```

Comment:
This may throw an error if `$('#search-form').find('.autocomplete-input').val()` is undefined.